### DISCUSSION TITLE
Vite + Remix on Azure

### DISCUSSION THREAD

**Discussion by ysaybuna1996** [remix-run/remix#9180] [Q&A]
Hello,

Is there an updated documentation that I can follow to successfully deploy Vite+Remix app on azure web app?

**Comment by amolotkov** [comment]
Have you had any luck with the deployment of Remix + Vite to Azure?

**Reply by ysaybuna1996** [reply]
> Have you had any luck with the deployment of Remix + Vite to Azure?

Unfortunately, no luck with azure. Instead I used container and uploaded the app to aws lightsail. Just make sure to expose your vite port to the container.
